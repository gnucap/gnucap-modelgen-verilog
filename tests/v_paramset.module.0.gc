
verilog

options trace

module ports(a, b, c)
  vsource #(.dc(3)) c(a, b);
endmodule
module ports(a, b)
  vsource #(.dc(2)) c(a, b);
endmodule

module main(1 2);
  ports p (1 0);
  ports p (2 0 0);
endmodule

main #() m(1 2);

print dc v(nodes)
dc
status notime
end
