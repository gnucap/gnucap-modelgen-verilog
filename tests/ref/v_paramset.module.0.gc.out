>>>>>module ports(a, b, c) 
>>>>>module ports(a, b) 
>>>>>module main(1 2); 
>>>>>print dc v(nodes) 
main.p: ports from top level
main.p discarded: not enough nodes, have 2 need 3

main.p: ports from top level
main.p: 1 candidate found for ports
main.p: ports from top level
main.p: ports from top level
main.p discarded: too many: requested=3 max=2
main.p: 1 candidate found for ports
ports.c: vsource from device_dispatcher
ports.c: 1 candidate found for vsource
ports.c: vsource from device_dispatcher
ports.c: 1 candidate found for vsource
>>>>>dc 
#           v(1)       v(2)      
 0.         2.         3.        
>>>>>status notime 
Gnucap   System status
iterations: op=0, dc=2, tran=0, fourier=0, total=2
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=2, subckt=0, model=0, total=2
dctran density=50.0%, ac density=50.0%
>>>>>end 
